frontend/lib/src/features/dashboard/dashboard.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:frontend/src/features/dashboard/components/account_card.dart';
+import 'package:frontend/src/models/account.dart';
+
+class Dashboard extends StatelessWidget {
+  Dashboard({
+    super.key,
+  });
+
+  /* This is the list of accounts pulled from BE */
+  /* NOTE: This is currently using mock data */
```

Comment:
```suggestion
  /* TODO (redempto): This is currently using mock data */
```

to find todos easier
you can also take a look at [todo tree ext](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree)